devtools/cli/distribution/jreleaser.yml

```diff
@@ -0,0 +1,47 @@
+project:
+  name: quarkus
+  description: CLI for Quarkus
+  longDescription: |
+    Create projects, manage extensions and perform build and development tasks for Quarkus,
+    the Kubernetes Native Java stack tailored for OpenJDK HotSpot and GraalVM.
+  website: https://quarkus.io
+  authors:
+    - Quarkus Community
+  license: Apache-2
+  java:
+    groupId: io.quarkus
+    artifactId: quarkus-cli
+    version : 11
+  tags:
+    - cli
+    - quarkus
+    - java
+
+release:
+  github:
+    owner: quarkusio
+    name: quarkus
+    tagName: "{{projectVersion}}"
+    overwrite: false
```

Comment:
The value of `overwrite` is `false` by default

Reply:
was using explicit values to make it easier for myself and others to understand what was going to happen...